### ISSUE TITLE
Single colour favicons are not always visible over generated background color

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-ios#2051]
### Description: 
If a favicon has a single colour, the generated background colour (based on colour in the favicon) makes the favicon itself not visible. This is very obvious on sites such as brianbondy.com

### Steps to Reproduce 
  1. Clean profile 1.14
  2. Enable rewards and fund the wallet
  3. Add few verified pubs to a-c table
  4. Send a one-time tip to a verified pubs
  5. Favicon isn't displayed for few verified pubs in a-c table, BR panel and tip sent thank you banner

**Actual result:** 
Favicon isn't displayed for few verified pubs in a-c table, BR panel and tip sent thank you banner

**Expected result:**
The favicon should be displayed for verified pubs in a-c table, BR panel and tip sent thank you banner

**Reproduces how often:** [Easily reproduced, Intermittent Issue]

Always
**Brave Version:** 
1.14

**Device details:** 
iPhone8

**Website problems only:**
- did you check with Brave Shields down? NA
- did you check in Safari/Firefox (WkWebView-based browsers)? NA
![IMG_0157](https://user-images.githubusercontent.com/38657976/69822648-e64eb400-122c-11ea-94c3-6fbdff3f3067.PNG)
![IMG_0159](https://user-images.githubusercontent.com/38657976/69822649-e64eb400-122c-11ea-94d4-4d4b7e8184ec.PNG)
![IMG_0160](https://user-images.githubusercontent.com/38657976/69822650-e6e74a80-122c-11ea-81d2-fa2ecd02eb48.PNG)
![IMG_0162](https://user-images.githubusercontent.com/38657976/69822651-e6e74a80-122c-11ea-9922-847c8ca4774d.PNG)

### Additional Information
Reproduced in play store version 1.13 also

cc: @brave/legacy_qa @jhreis

**Comment by kylehickinson** [comment]
This ticket doesn't actually have anything to do with Rewards related favicons, it affects all favicons in the app. The favicon is there, but we generated a background color for favicons based on dominate color, and in this case, brianbondy.com's favicon is all 1 color, which will affect its visibility:

![Simulator Screen Shot - iPhone 11 Pro - 2019-11-29 at 16 06 17](https://user-images.githubusercontent.com/529104/69890052-2f9d1300-12c2-11ea-9e5f-4a1fdef1d335.png)

**Comment by srirambv** [comment]
Verification passed on iPhone XR with iOS 13.5 running 1.18(20.06.16.08)
- Verified publishers favicon is shown correctly in BR panel
- Verified publishers favicon is shown correctly in AC list
- Vierified publishers favicon is shown correctly in Tabs overview
------------------
Verification passed on iPhone 7+  with iOS 13.4.5 running 1.18(20.06.16.08)
- Verified publishers favicon is shown correctly in BR panel
- Verified publishers favicon is shown correctly in AC list
- Vierified publishers favicon is shown correctly in Tabs overview
------------------
Verification passed on iPad Pro with iOS 12.4.5 running 1.18(20.06.16.08)
- Verified publishers favicon is shown correctly in BR panel
- Verified publishers favicon is shown correctly in AC list
- Vierified publishers favicon is shown correctly in Tabs overview
------------------
